Project.toml

```diff
@@ -21,7 +21,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 julia = "≥ 1.0.0"
 CDDLib = "≥ 0.5.2"
 Distributions = "≥ 0.20.0"
-Documenter = "^0.22.6"
+Documenter = "≥ 0.23.0"
```

Comment:
Should we use `Documenter = "^ 0.23.0" ? To make sure that LazySets builds do not start failing suddenly due to a new major release of Documenter, which may include breaking changes. Related: https://juliadocs.github.io/Documenter.jl/dev/man/hosting/#.travis.yml-Configuration-1

Reply:
We do not do that for other packages as well, so I tend not to. It is usually easy to detect such cases.

Reply:
Ok.